docs/data/toolpad/core/introduction/installation.md

```diff
@@ -4,47 +4,47 @@ title: Toolpad Core - Installation
 
 # Installation
 
-## Manual Installation
+<p class="description">Learn how to install Toolpad Core in your local environment.</p>
+
+## Manual installation
```

Comment:
Sentence case

Reply:
Can we make vale catch these?

Reply:
We could use https://vale.sh/docs/topics/styles/#capitalization. Now,

- There are valid uses of title cases when we reference components, e.g. https://mui.com/toolpad/core/react-app-provider/#next-js-app-router seems right, it's how Next.js labels it. 
- Vale CI is broken: #4347.